testing-guidance/workflow-tests/android/android-lkg-workflows.md

```diff
@@ -0,0 +1,77 @@
+# Android LKG Workflow Tests
+
+Testing in this area should focus on the different ways O3DE can be built and deployed into an android device and the launch of a default level successfully.
+
+## General Android Docs
+* [O3DE User Guide: Android Support for O3DE](https://www.o3de.org/docs/user-guide/platforms/android/)
+
+## Common Issues to Watch For
+
+- Binary and assets are located at their correct place for each packaging mode.
+- The most common packaging mode used during Android development is VFS (Virtual File System), be aware of network connection errors from the mobile device to Asset Processor running on PC.
+- Rendering artifacts when running default level on Android devices.
+- Low framerates when running default level on Android devices. Look out for spamming error/warnings in the Android log.
+- Asserts and Errors that appear in the Android Log while running default level.
+- Immediate crashes when launching the app on the Android device.
+- Crashes that occur after some time leaving the app running.
+
+## Workflows
+
+### Area: Build android in different packaging modes
+
+**Project Requirements**
+* Create a new default project (`DefaultProject`)
+* Modify `o3de\Registry\AssetProcessorPlatformConfig.setreg` to generate android assets in the cache. Remove `//` the following android line:
```

Comment:
We shouldn't recommend to users to modify the "engines" AssetProcessorPlatformConfig.setreg file for generating Android assets. That file is sourced controlled and has the potential to be accidentally submitted.
Furthermore it keeps `git` from seeing the workspace as clean when making modifications to it

Instead we should recommend that the user add a `<project-root>/user/asset_processor.setreg` with the following content to enable AP processing for Android
```
{
    "Amazon": {
        "AssetProcessor": {
            "Settings": {
                "Platforms": {
                    "android": "enabled"
                }
            }
        }
    }
}
```

Reply:
Should that be `<project-root>/user/Registry/asset_processor.setreg`?

Maybe worth noting the distinction between the "user" vs. "project" scopes:
`<project-root>/user/Registry/asset_processor.setreg` - User scope, only affects that user's local cache, and would NOT be checked into source control.
`<project-root>/Registry/asset_processor.setreg` - Project scope, useful if making the change for the team, and would be checked into source control.

Reply:
Yes, that should be `<project-root>/user/Registry/asset_processor.setreg`.
I'll edit the comment.

Reply:
Now for the distinction between user and project scopes, that is documented in the Settings Registry documentation.

Reply:
Excellent point! I'll test locally using the "user" version (it should work, but I'll check). I'll make the changes after confirming it's working. I'll test doing the same for the bootstrap settings.

Reply:
I tested this. I couldn't use `<project-root>/user/Registry/asset_processor.setreg` file because the first time Asset Processor is executed that file gets overwritten and it puts other content, that's probably a bug where if the file already exists it should merge it instead of replacing it.

To workaround that I just used another filename `<project-root>/user/Registry/AssetProcessor.setreg` and that worked. I'll use that in the document for now.

Reply:
Yeah, the AP just writes out a certain subsets of settings over asset_processor.setreg, so it doesn't perform any merging.
Having a different name works for any user.
The name of the file itself isn't important as long as it exist within the user/Registry directory, so that it is picked up by the folder merge.